MatrixSDK/Data/MXRoom.m

```diff
@@ -1311,6 +1382,132 @@ - (MXHTTPOperation *)reportEvent:(NSString *)eventId
 }
 
 
+#pragma mark - Message order preserving
+/**
+ Make sure that `block` will be called in the order expected by the end user.
+
+ @param localEvent the local echo event corresping to the event being sent.
```

Comment:
- typo "corresping"
- What do you think to consider only the local event id here?

Reply:
> What do you think to consider only the local event id here?
Yeah, possibly. ```localEvent``` could be interesting in the future, or not. Anyway, given it is a private method, we will be free to change it in the future.